src/docs/sanapi/available-metrics.md

```diff
@@ -22,13 +22,18 @@ unified way.
 
 Supported assets:
 
-| Bitcoin            | Ethereum           | ERC-20             | Ripple             | EOS                | EOS tokens         | Binance Chain      |
-| ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
-| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Bitcoin            | Ethereum           | ERC-20             | Ripple             | Binance Chain      |
+| ------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 
+| EOS                | EOS tokens         | Bitcoin Cash       | Litecoin           |
+| ------------------ | ------------------ | ------------------ | ------------------ |
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 
 ## timeseriesData
 
+Timeseries data is is a sequence taken at successive equally spaced points in time. For every point the API provides a `datetime` and a `value` fields.
```

Comment:
Timeseries data is a sequence taken at successive equally spaced points in time. For every point the API provides a `datetime` and a `value` field.